datalab/mlalpha/_cloud_models.py

```diff
@@ -95,11 +89,36 @@ def delete(self, model_name):
       full_name = ('projects/%s/models/%s' % (self._project_id, model_name))
     return self._api.projects().models().delete(name=full_name).execute()
 
+  def list(self, count=10):
+    """List models under the current project in a table view.
+
+    Args:
+      count: upper limit of the number of models to list.
+    Raises:
+      Exception if it is called in a non-IPython environment.
+    """
+    import IPython
+    data = [{'name': model['name'], 
+             'defaultVersion': model['defaultVersion']['name'].split('/')[-1]
+                 if 'defaultVersion' in model else None}
+            for _, model in zip(range(count), self)]
```

Comment:
so model is always equal to self? This list comprehension is complicated enough to just do it in a loop.